lab_2_retrieval_w_bm25/main.py

```diff
@@ -33,6 +50,25 @@ def remove_stopwords(tokens: list[str], stopwords: list[str]) -> list[str] | Non
 
     In case of corrupt input arguments, None is returned.
     """
+    if not (
+            isinstance(tokens, list) and
+            all(isinstance(token, str) for token in tokens)
+    ):
+        return None
+    if not (
+            isinstance(stopwords, list) and
+            all(isinstance(word, str) for word in stopwords)
+    ):
+        return None
+    if len(tokens) == 0 or len(stopwords) == 0:
```

Comment:
try thruthy pattern: https://stackoverflow.com/questions/39983695/what-is-truthy-and-falsy-how-is-it-different-from-true-and-false

Reply:
done